### ISSUE TITLE
Stoplight Studio/Doc does support array[string] with enumerations. 

### ISSUE THREAD

**Issue by mrum-cuscal** [stoplightio/studio#301]
Existing json swagger supports an array[string] with enums.

[Stoplight Issue.docx](https://github.com/stoplightio/studio/files/4069214/Stoplight.Issue.docx)

**Comment by rossmcdonald** [comment]
@mrum-cuscal Thanks for bringing this to our attention. It looks like enums set on array[string] values are applied to the array instead of the string property. Is that the issue you are seeing as well?

For example:

<img width="605" alt="image" src="https://user-images.githubusercontent.com/4060903/72534949-50eb9b80-383d-11ea-9078-f0858526504c.png">

Results in:

```
                "schema": {
                  "type": "array",
                  "enum": [
                    "orange",
                    "yellow",
                    "red"
                  ],
                  "items": {
                    "type": "string"
                  }
                },
```

When it should be:

```
                "schema": {
                  "type": "array",
                  "items": {
                    "type": "string"
                    "enum": [
                      "orange",
                      "yellow",
                      "red"
                      ]
                    }
                  },
```

**Comment by lottamus** [comment]
We've just released Studio Desktop [v1.12](https://github.com/stoplightio/studio/releases/tag/v1.12.0) which includes a fix for this issue. Please feel free to re-open if you're still seeing the issue after upgrading.
